### PR TITLE
fix: div by zero error

### PR DESCRIPTION
**Pull request by teamchong**
Fixed divide by zero error on market recoil state

## Checklist
- [X] Ran Linting
- [X] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [X] Added an entry to the `CHANGELOG.md` file.
- [X] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Review by yayay927 — CHANGES_REQUESTED** [review]
The logo size seems abnormal (much bigger) from what we usually have. Can you double check? And can you also double check with backend about the 6 months delayed data?
截圖 2022-09-11 下午11 44 43

**Comment by teamchong** [comment]
> The logo size seems abnormal (much bigger) from what we usually have for previous BDs. Can you double check? And can you also double check with backend about the 6 months delayed data? 截圖 2022-09-11 下午11 44 43

thanks! I will address this in another PR

File: CHANGELOG_PERSISTENCE.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Bug fixes
+
+- Fixed divide by zero error on market recoil state
```
Comment (yayay927) [inline CHANGELOG_PERSISTENCE.md:5]: @teamchong please add the issue link

**Review by yayay927 — CHANGES_REQUESTED** [review]

**Review by yayay927 — CHANGES_REQUESTED** [review]
@teamchong I've merged pr #1001 , please pull the latest branch change and push again. Thx

**Pull request merged**